include/menu.h

```diff
@@ -0,0 +1,28 @@
+#ifndef MENU_H
+#define MENU_H
+
+#include <stdio.h>
+#include <stdlib.h> // needed for function exit, dynamic memory mgmt
+#include <ctype.h> // needed for function tolower
+
+
+struct menu_s {
+    char symbol; 
+    char *text; 
+    };
+typedef struct menu_s menu_t;
+
+
+//needed here in .h??? - pls give feedback in code review
+void show_menu();
+void parse_input();
+
+
+/** example to show how to do simple debugging
+ * should be brought to start of main.c???
+ * the way to comment here also here a test for doxygen - very very first draft
+   */
+#define D 1 //DEBUG_LEVEL
+
+
+#endif
```

Comment:
Wolltest du hier noch ein #if einbauen? Oder meintest du mit IF Abfragen hauptsächlich im main.c?

Reply:
dieses EndIf ist vom define drüber - ich meinte nur, dass wir in allen c-Dateien dann die ifs (#ifs) verwenden können um was auszulassen

Reply:
Aso ja stimmt, sry mein fehler dass kommt ja noch vom #ifndef :sweat_smile: